src/openrct2-ui/windows/StaffList.cpp

```diff
@@ -531,7 +531,8 @@ void window_staff_list_invalidate(rct_window* w)
     {
         window_staff_list_widgets[WIDX_STAFF_LIST_UNIFORM_COLOUR_PICKER].type = WWT_COLOURBTN;
         window_staff_list_widgets[WIDX_STAFF_LIST_UNIFORM_COLOUR_PICKER].image = SPRITE_ID_PALETTE_COLOUR_1(
-                                                                                     (uint32_t)staff_get_colour(tabIndex))
+                                                                                     static_cast<uint32_t>(
+                                                                                         staff_get_colour(tabIndex)))
```

Comment:
This looks very silly like this. It's better to use an intermediary variable, I'd say.